cli/bin/grain.js

```diff
@@ -25,9 +25,18 @@ function onRespawn(ready, proc, argv) {
   const path = require('path');
 
   let program = require('commander');
-  let compile = require('./compile.js');
-  let run = require('./run.js');
-  let lsp = require('./lsp.js');
+  // Workaround to defer loading of the grain runtime until the memory settings have been parsed
+  let imports = {
```

Comment:
Could we name this something else, like `actions`? Seeing `imports` here makes me think it's a WebAssembly import object.

Reply:
Done! Naming is one of the hardest problems in computer science